src/languages/en.js

```diff
@@ -328,6 +328,7 @@ export default {
         payerOwesAmount: ({payer, amount}) => `${payer} owes ${amount}`,
         noReimbursableExpenses: 'This report has an invalid amount',
         pendingConversionMessage: "Total will update when you're back online",
+        threadReportName: ({payee, comment}) => `${payee} request${comment ? ` for ${comment}` : ''}`,
```

Comment:
This should be `<formattedAmount> request for <comment>`

Reply:
haha gotcha I wasn't 100% sure based on the doc b/c I'd copied it as `$X request for $comment` and wasn't sure if X was meant to be amount or the person making the request 😂